src/sentry/rules/processing/processor.py

```diff
@@ -405,7 +405,7 @@ def apply_rule(self, rule: Rule, status: GroupRuleStatus) -> None:
                 "post_process.process_rules.triggered_rule",
                 extra={
                     "rule_id": rule.id,
-                    "payload": state,
+                    "payload": state.__dict__,
```

Comment:
You could also implement `repr` on the `EventState`, so it will be easy to understand whenever converted to a string

Reply:
Right, I was more thinking that if we implement `repr` then any time we try and print this object it'll work fine